cirq-google/cirq_google/engine/engine.py

```diff
@@ -126,6 +126,28 @@ def copy(self) -> 'EngineContext':
     def _value_equality_values_(self):
         return self.proto_version, self.client
 
+    def serialize_program(
```

Comment:
I find the name "program" confusing here. Since it's moving to public visibility, what do you think of making it something like `serialize_circuit` and let the weird naming in `create_program` be its own problem?

Reply:
The term "program" is fairly well-established, for example it is used in all the [`cirq.Sampler`](https://github.com/quantumlib/Cirq/blob/master/cirq-core/cirq/work/sampler.py#L34) methods, so I'm inclined to leave this, but I could be persuaded.